src/main/java/com/google/api/codegen/viewmodel/CallingForm.java

```diff
@@ -27,15 +27,17 @@
   //
   // [Method signature type][Request pattern][Response pattern][Idiomatic pattern]
 
-  Request, // used by: java nodejs py
+  Request, // used by: java nodejs py php
   RequestAsync,
   RequestAsyncPaged, // used by: nodejs
   RequestAsyncPagedAll, // used by: nodejs
-  RequestPaged, // used by: java py
-  RequestPagedAll, // used by: py
-  RequestStreamingBidi, // used by: nodejs py
-  RequestStreamingClient, // used by: nodejs py
-  RequestStreamingServer, // used by: nodejs py
+  RequestPaged, // used by: java py php
```

Comment:
For consistency, let's keep the language comments in alphabetical order.

Reply:
Done.